status-handler.js

```diff
@@ -60,6 +62,8 @@ class StatusHandler {
       Status.findOne(ev)
     ]);
 
+    await this._dispatchWebhook(data);
```

Comment:
this now as an `event` argument so I imagine this is 
```suggestion
    await this._dispatchWebhook('build_started', data);
```
?

Reply:
Errors for this should also not throw and kill rest of execution so wrap this in a `try catch` so things will continue to process and just log the error